generators/app/templates/.gitignore

```diff
@@ -18,39 +18,32 @@ Thumbs.db
 
 # Compiled Files
 # ---------------------------
-/dist/<% if (platform == 'craft') { %>
+/dist/
 
 # Craft
 # ---------------------------
 /craft/plugins/*
 /craft/rev-manifest.json
-/craft/storage/
+/craft/storage/*
 
 # Craft Public
 # ---------------------------
-/public/.htaccess
-/public/assets/
-/public/assets/**/_*
-/public/images/
-/public/scripts/
-/public/styles/
-/public/uploads/
-/public/web.config<% } %>
+/public/*
+!/public/index.php
+!/public/*.txt
+!/public/*.png
+!/public/*.ico
 
 # Package Managers
 # ---------------------------
-/.bundle
 /vendor/
-bower_components
-node_modules
-npm-debug.log
+/node_modules/
 
 # Temp
 # ---------------------------
 .cache
-.sass-cache
-/log/*.log
-/logs/*
+*.log
+/logs/
```

Comment:
Interesting. I swear I've encountered this on a non-ruby project. I haven't used ruby-sass on a project in over a year probably